zipkin-ui/js/component_data/spanNode.js

```diff
@@ -88,6 +88,32 @@ function keyString(id, shared = false, endpoint) {
   return `${id}-${endpointString}`;
 }
 
+function nodeByTimestamp(a, b) {
```

Comment:
:(

Is the code completely duplicated between the ui and lens? Given that the core modules should be identical, could we symlink them to avoid drift?